CanvasCore/CanvasCore/Push Notifications/Model/CommunicationChannel.swift

```diff
@@ -25,6 +25,8 @@ enum CommunicationChannelType: String, CustomStringConvertible {
     case twitter = "twitter"
     case yo = "yo"
     case push = "push"
+    case slack = "slack"
+    case unknown = "unknown"
```

Comment:
Now that I've looked at the code a little more, I don't think we need an unknown case. Mind removing this?